aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -711,37 +714,109 @@ Resources:
   # End AWS Cloudfront Resources #
   ################################
 
+  #################################
+  # START Github Actions IAM Role #
+  #################################
+
+  GithubDeployerUserPolicy:
```

Comment:
We don't want GitHub Actions to have permissions to define the Role/Policy it assumes to provision a marketing site Stack, right? I can move this over to the [in-progress update of the IAM Stack](https://github.com/code-dot-org/code-dot-org/pull/65584/files).